index.html

```diff
@@ -0,0 +1,102 @@
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap">
+    <title>Portfolio</title>
+  
+</head>
+<body>
+
+  <header>
+    <appbar class="appbar">
+        
+        <nav class="appbar  navbar navbar-light fixed-top">
+          <div class="container-fluid">
+            <div class="nav-container">
+            <a class="navbar-brand" href="#">My Logo</a>
+            </div>
+            <button class="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+              
+              <div class="offcanvas-body">
+                <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                  <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">Portfolio</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">Contact</a>
+                  </li>
+                                 
+                </ul>
+            
+              </div>
+            </div>
+          </div>
+        </nav>
+    </appbar>
+</header>
+
+  <div class="main">
+  <section class="mobile">
+      <h1 class="mobile heading">I’m Mariane
+      Glad to see <br> you!</h1>
+      <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <h5 id="heading5">LET’S CONNECT</h5>
+
+      <section class="icon">
+        <ul>
+         <li><a href="#"><img class="icon icon1" id="twitter"  src="sources/Vector4.png"></a></li>    
+         <li><a href="#"><img class="icon icon2" id="linkedin" src="sources/Linkedin icon.png"></a></li>     
+         <li><a href="#"><img class="icon icon3" id="medium"   src="sources/Vector5.png"></a></li>        
+         <li><a href="#"><img class="icon icon4" id="github"  src="sources/Vector1.png"></a></li>        
+         <li><a href="#"><img class="icon icon5" id="peace"  src="sources/Vector3.png"></a></li>   
+      </ul>
+    </section>  
```

Comment:
- [ ] Good job using semantic tags `ul` and `li` to structure this section. While doing so I think a bug was introduced. Kindly update the alignment of your social links to match the Figma design.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/205439242-2ac5d25c-a282-49ac-acc5-4720aa43b219.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/205439294-5b865d05-d173-4b4c-b262-08737494e996.png" width="300">|